framework/field_functions/field_function_grid_based.cc

```diff
@@ -315,7 +315,7 @@ FieldFunctionGridBased::ExportMultipleToVTK(
       point_data->AddArray(point_array);
       cell_data->AddArray(cell_array);
     } // for component
-  }   // for ff_ptr
+  } // for ff_ptr
```

Comment:
@wdhawkins @andrsd it is not hte first time that we come cross the issue of comment alignments in nested loops. It seems that a clang default is to not do what we do. Should we abandon our alignment request. I personally prefer when the comments are not all on the same column

Reply:
IIRC, the problem is "caused" by clang-format-14. The newer versions may have more flags to control these cases.

I, personally, don't like these comments at all. Any modern editor can find matching curly brackets, but some people find those useful. This is one of the downsides of using automatic code formatters...